src/programs/refine3d/refine3d.cpp

```diff
@@ -979,7 +982,7 @@ bool Refine3DApp::DoCalculation()
 	temp_image_local.Allocate(input_stack.ReturnXSize(), input_stack.ReturnYSize(), true);
 	projection_image_local.Allocate(binned_image_box_size, binned_image_box_size, false);
 	if (ctf_refinement && high_resolution_limit <= 20.0) binned_image.Allocate(binned_image_box_size, binned_image_box_size, false);
-	if (global_search)
+	if (global_search_local || local_global_refine)
```

Comment:
what does `local_global_refine` mean? does the `local` refer to thread locality? If so maybe `this_thread` instead?

Reply:
this is a bool that defines whether we are running in the special mode that allows a mixture or global and local refinement.  It's actually one of the program inputs, and has been in the code since cisTEM 1.0.0

Reply:
yeh I guess that might be the case after I wrote the comment. I guess I still find it confusing that `local` is used to mean two different thigns in this file..